src/AddTodoForm.js

```diff
@@ -18,16 +19,12 @@ function AddTodoForm({ onAddTodo }) {
   return (
     <div>
       <form onSubmit={handleAddTodo}>
-        <label htmlFor="todoTitle">
-          Title:
-          <input
-            type="text"
-            name="title"
-            id="todoTitle"
-            value={todoTitle}
-            onChange={handleTitleChange}
-          />
-        </label>
+        <InputWithLabel
+          handleTitleChange={handleTitleChange}
+          todoTitle={todoTitle}
+        >
+          Title:{" "}
+        </InputWithLabel>
         <button type="submit">Add</button>
```

Comment:
💯

Reply:
If you remove the `event.preventDefault` in `handleAddTodo`, the `input` field will always receive focus after you add a new to do item b/c the page will submit. Check it out!! 😸